README.md

```diff
@@ -1,15 +1,47 @@
-# bfx-api-node-core
+# Bitfinex Modular WSv2 API Library for Node.JS
 
 [![Build Status](https://travis-ci.org/bitfinexcom/bfx-api-node-core.svg?branch=master)](https://travis-ci.org/bitfinexcom/bfx-api-node-core)
 
-Modular Bitfinex Node.JS API library as an alternative to `bitfinex-api-node`, supporting a plugin system.
+Modular Bitfinex Node.JS API library as an alternative to `bitfinex-api-node`, supporting a plugin system. Connection instances are POJOs as opposed to the
+WSv2 class instances returned by `bitfinex-api-node` and are manipulated in a
+functional style. A connection pool manager is also provided for multiplexing.
+
+### Features
+
+* POJO connection instances
+* Multiplexing connection pool manager
+* Plugin system for extending the default events
+
+### Installation
+
+```bash
+npm i --save bfx-api-node-core
+```
+
+### Quickstart
 
-### Example: Subscribing to candles
 ```js
-'use strict'
+const { Manager, initState } = require('bfx-api-node-core')
+
+// Create a Manager instance with an internal connection pool, and add a
+// connection to the pool
+const m = new Manager({ transform: true })
+const managedConnection = m.openWS()
+
+// Alternatively, create & open a single connection yourself
+const connection = initState({ transform: true })
 
-process.env.DEBUG = '*'
+// do something with connections, see below for examples
+```
+
+### Docs
+
+See `docs/manager_docs.md` for the Manager class documentation, and `docs/ws2_funcs.md` for documentation on the functions available for manipulating a connection instance.
```

Comment:
No biggie but would be nice to get a relative href link in here `[See manager_docs.md](docs/manager_docs.md)`

Reply:
I'll update this here & in the other PRs